src/main/kotlin/study/step3/car/Car.kt

```diff
@@ -0,0 +1,15 @@
+package study.step3.car
+
+class Car() {
```

Comment:
```suggestion
class Car {
```
빈 생성자의 경우 생략이 가능합니다.